app/templates/teachers/teacher-student-view.jade

```diff
@@ -0,0 +1,220 @@
+extends /templates/base-flat
+
+block page_nav
+  include ../courses/teacher-dashboard-nav.jade
+
+block content
+
+  if !view.user
+    div Loading
+  else
+    .container
+      +breadcrumbs
+      if !me.isAnonymous() && me.isTeacher()
+      - var isOwner = view.classroom ? view.classroom.get('ownerID') === me.id : false;
+        // - var validStudent = ...  
+        // load all students in this class
+        // check if this student is in the class
```

Comment:
Cleaned up -- a lot of comments were leftover from early stages where I wasn't sure what I was doing :)